docs/lua-modding/packing-a-lua-mod.mdx

```diff
@@ -0,0 +1,12 @@
+---
+sidebar_position: 10
+---
+
+# Packing a Lua Mod
+Packing a Lua-only mod is super simple.
```

Comment:
I don't think we need this page, the lua mod structure describes the structure pretty well, and packaging instructions are mod manager dependent